app/views/welcome/_other_sidebar.html.erb

```diff
@@ -14,6 +15,17 @@
 		<dt>Phone</dt>
 		<dd><%= number_to_phone @person.phone_mobile %></dd>
 
+		<%- if @current_user.person_type == "Participant" -%>
+		  <dt><%= link_to "Acknowledgement of Intake Form", participant_signup_intake_form_path %></dt>
```

Comment:
It might be helpful to separate the section heading from the edit link, something like
```ruby
<dt>Intake Form (<%= link_to "Edit Responses", participant_signup_intake_form_path %>)</dt>
```